apps/console/src/features/applications/utils/application-management-utils.ts

```diff
@@ -248,13 +248,28 @@ export class ApplicationManagementUtils {
             doc = parser.parseFromString(strXML, "text/xml");
         }
 
+        const childNodeArray = doc.getElementsByTagName("IDPSSODescriptor")[0]?.childNodes;
```

Comment:
we can try `HTMLCollectionOf<HTMLStyleElement>` or `NodeList` as the type

Reply:
Fixed with 1d3ef8fd4f615de27060c959ea91a56ca1f3158f